bin/setup.bat

```diff
@@ -15,6 +15,10 @@ goto finally
 
 :setup_jruby
 REM setup_java()
+if not defined JAVA_HOME IF EXIST %ProgramData%\Oracle\java\javapath\java.exe (
+    for /f "tokens=2 delims=[]" %%a in ('dir %ProgramData%\Oracle\java\javapath\java.exe') do @set JAVA_HOME=%%a
```

Comment:
@StefanScherer Can we log/echo that we are deducing the `JAVA_HOME` from this location?